gatsby/src/templates/situation/page.tsx

```diff
@@ -2,13 +2,28 @@ import React from 'react';
 import { graphql } from 'gatsby';
 import { IQuery } from 'graphql-types';
 import SituationDetail from '@/components/situation-detail/situation-detail';
-
+import ContentBox from '@/components/content-box';
+import LinkList from '@/components/link-list';
+import Container from '@/components/container';
 interface IProps {
   data: IQuery;
 }
 
 const Page: React.FC<IProps> = ({ data }) => {
-  return <SituationDetail situation={data.situation} />;
+  return (
+    <>
+      <SituationDetail situation={data.situation} />
+
+      <Container>
+        <ContentBox title="Podobná témata" boldedTitleCount={1} variant="blue">
```

Comment:
Nadpisy etc. by měly být lokalizované. Příkladové query (nejprve je potřeba v createPages přidat do situation contextu langcode): 
`contentTitle: translation(source: {eq:"related"}, langcode: {eq: $langCode}){
    target
  }`
```suggestion
        <ContentBox title={contentTitle.target} boldedTitleCount={1} variant="blue">
```

Reply:
Pokud na tom nemáš v plánu dělat, tak to prosím přidej do Trello TODOs s linkem na tenhle komentář.